lib/gooddata/lcm/actions/collect_segments.rb

```diff
@@ -24,21 +24,18 @@ class CollectSegments < BaseAction
 
       class << self
         def call(params)
-          client = params.gdc_gd_client
-
-          domain_name = params.organization || params.domain
-          domain = client.domain(domain_name) || fail("Invalid domain name specified - #{domain_name}")
-          domain_segments = domain.segments
-          params.gdc_logger.info("Domain segments: #{domain_segments}")
+          data_product = params.data_product
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)